aireview/README.md

```diff
@@ -1,2 +1,54 @@
 # AIReview
-Pull request review agent
\ No newline at end of file
+Pull request review agent
```

Comment:
**documentation**: Missing installation command for required dependencies. Add `pip install -r requirements.txt` or specific package installation instructions